src/javascript/app_2/Modules/Page404/Components/Page404.jsx

```diff
@@ -0,0 +1,27 @@
+import React          from 'react';
+import { Link }       from 'react-router-dom';
+import { routes }     from 'Constants/index';
+import { localize }   from '_common/localize';
+import { Icon404 }    from './Icon404.jsx';
+import Button         from '../../../App/Components/Form/button.jsx';
+import ErrorBox       from '../../../App/Components/Elements/ErrorBox';
+
+const Page404 = () => (
+    <div className='page-error-container'>
+        <ErrorBox
+            header={localize('Page not found')}
+            icon={<Icon404 />}
+            message={localize('Sorry, we couldn\'t find the page you are looking for.')}
+        >
+            <Link to={routes.trade}>
```

Comment:
@zakibinary 
Since the `<Link>` tag is transformed to `<a>` and according to the [HTML5 spec document form W3C](http://w3c.github.io/html/textlevel-semantics.html#the-a-element), a `<button>` tag is not allowed to nest inside an `<a>` tag in HTML5. It's better to customize the `<Link>` component and add it the appropriates CSS classes.

Also, we have to fix it in [empty_statement_message](https://github.com/binary-com/binary-static/blob/master/src/javascript/app_2/Modules/Statement/Components/empty_statement_message.jsx#L23)